docs/js/guides/bas-external-system.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Connecting to external system from the Business Application Studio (BAS) with the SAP Cloud SDK for JavaScript / TypeScript
+hide_title: false
+hide_table_of_contents: false
+sidebar_label: Connecting to External system form BAS
+description: This article describes how the SDK helps to connect to external systems from the Business Application Studio.
+keywords:
+- sap
+- cloud
+- sdk
+- proxy
+- connectivity
+- Business Application Studio
+- cloud-foundry
+- JavaScript
+- TypeScript
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+## What is the Business Application Studio (BAS)? ##
+
+The Business Application Studio (BAS) is a development environment offered as a service on Cloud Foundry (CF).
+You can simply subscribe to the BAS and from there quickly start developing without installing node, git, Visual Studio Code or other tools.
+From the look and feel it is very similar to Visual Studio Code, which is no surprise since it is based on [Eclipse Theia](https://theia-ide.org/), the open source version of Visual Studio Code.
+
+However, SAP added a few useful features to the BAS. 
+You can connect the BAS to your CF account.
+This allows you to reach behind a Cloud Connector from your locally running BAS application without deploying to CF.
+This is a very useful feature for developers because the cycle to test something is much quicker if you can do it locally.
+
+<img alt="Connecting to external On-Premise systems" src={useBaseUrl('img/cloud-connector.png')} />
+
+## Technical Background <a name="background"></a> ##
+
+Companies do not expose their On-Premise SAP S/4HANA systems to the internet.
+They are only reachable via a Cloud Connector (CC) attached to a CF account.
```

Comment:
This is not the first time Cloud Connector is mentioned. Maybe you want to move the abbreviation up?